helm/hades/templates/hades-operator-deployment.yaml

```
@@ -0,0 +1,31 @@
+          resources:
+            limits:
+              cpu: {{ .Values.hadesOperator.resources.limits.cpu | quote }}
+              memory: {{ .Values.hadesOperator.resources.limits.memory | quote }}
+            requests:
+              cpu: {{ .Values.hadesOperator.resources.requests.cpu | quote }}
+              memory: {{ .Values.hadesOperator.resources.requests.memory | quote }}
+          ports:
+            - containerPort: {{ .Values.hadesOperator.service.targetPort }}
+          env:
+            - name: WATCH_NAMESPACE
+              value: "hades"
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)